app/code/Magento/Dhl/etc/module.xml

```diff
@@ -6,6 +6,5 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_Dhl" >
-    </module>
+    <module name="Magento_Dhl" />
```

Comment:
`Dhl` module depends on `Config` and `Shipping` (quick look).

Reply:
Space removed